test/views/DeclarationDetailsViewSpec.scala

```diff
@@ -46,16 +48,30 @@ class DeclarationDetailsViewSpec extends UnitViewSpec with GivenWhenThen with In
 
   private val submission = Submission("id", "eori", "lrn", Some(mrn), Some("ducr"), Seq.empty)
 
+  private val actionIdDiscriminant = "id2"
+
   private val acceptedNotification = Notification("id", mrn, now, ACCEPTED, Seq.empty)
-  private val clearedNotification = Notification("id", mrn, now, CLEARED, Seq.empty)
+  private val dmsqry1Notification = Notification("id1", mrn, now, QUERY_NOTIFICATION_MESSAGE, Seq.empty)
+  private val dmsqry2Notification = Notification(actionIdDiscriminant, mrn, now, QUERY_NOTIFICATION_MESSAGE, Seq.empty)
   private val dmsdocNotification = Notification("id", mrn, now, ADDITIONAL_DOCUMENTS_REQUIRED, Seq.empty)
   private val dmsctlNotification = Notification("id", mrn, now, UNDERGOING_PHYSICAL_CHECK, Seq.empty)
-  private val rejectedNotification = Notification("id", mrn, now, REJECTED, Seq.empty)
+  private val dmsrejNotification = Notification("id", mrn, now, REJECTED, Seq.empty)
+
+  // Since the notification list is reverse-ordered (more to least recent) in TimelineEvents...
```

Comment:
"(**most** to least recent" :slightly_smiling_face:

Reply:
:upside_down_face: